<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vs2015_qt_signalslotClass</class>
 <widget class="QWidget" name="vs2015_qt_signalslotClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vs2015_qt_signalslot</string>
  </property>
  <widget class="QPushButton" name="pushButtonTextChange">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTextClearShow">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTextChange">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTextClearShow">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="vs2015_qt_signalslot.qrc"/>
 </resources>
 <connections/>
</ui>
